src/main/java/com/jsoniter/IterImplForStreaming.java

```diff
@@ -382,92 +382,127 @@ public static int updateStringCopyBound(final JsonIterator iter, final int bound
         }
     }
 
+    public static boolean[] cover_readStringSlowPath = new boolean[34];
+
     public final static int readStringSlowPath(JsonIterator iter, int j) throws IOException {
         boolean isExpectingLowSurrogate = false;
         for (;;) {
+            cover_readStringSlowPath[0] = true;
             int bc = readByte(iter);
             if (bc == '"') {
+                cover_readStringSlowPath[1] = true;
                 return j;
             }
             if (bc == '\\') {
+                cover_readStringSlowPath[2] = true;
                 bc = readByte(iter);
                 switch (bc) {
                     case 'b':
+                        cover_readStringSlowPath[3] = true;
                         bc = '\b';
                         break;
                     case 't':
+                        cover_readStringSlowPath[4] = true;
                         bc = '\t';
                         break;
                     case 'n':
+                        cover_readStringSlowPath[5] = true;
                         bc = '\n';
                         break;
                     case 'f':
+                        cover_readStringSlowPath[6] = true;
                         bc = '\f';
                         break;
                     case 'r':
+                        cover_readStringSlowPath[7] = true;
                         bc = '\r';
                         break;
                     case '"':
+                        cover_readStringSlowPath[8] = true;
```

Comment:
Shouldn't you add `cover_readStringSlowPath[8] = (bc == '"');` here as well, since there is no break statement?

Reply:
Since the previous statement has a break it can't fall through to this one, so it will only happen for bc == '"' anyway.

Reply:
Right, that's true.